backend/tracim_backend/tests/library/test_workspace.py

```diff
@@ -155,3 +157,29 @@ def test__unit__workspace_deletion__ok__nominal_case(self) -> None:
         assert business_workspace.is_deleted is True
         assert business_workspace.label != "business"
         assert business_workspace.label.startswith("business-deleted-")
+
+    def test_unit__create_workspace_same__error__same_workspace_name_unallowed(self):
+        admin = self.session.query(User).filter(User.email == "admin@admin.admin").one()
+        wapi = WorkspaceApi(session=self.session, current_user=admin, config=self.app_config)
+        wapi.create_workspace(label="business", save_now=True)
+        with pytest.raises(WorkspaceLabelAlreadyUsed):
+            wapi.create_workspace(label="business", save_now=True)
+
+    def test_unit__rename_workspace_same_name_same_file__ok__nominal_case(self):
+        admin = self.session.query(User).filter(User.email == "admin@admin.admin").one()
+        wapi = WorkspaceApi(session=self.session, current_user=admin, config=self.app_config)
+        workspace1 = wapi.create_workspace(label="business", save_now=True)
+        modified_datetime = workspace1.updated
+        wapi.update_workspace(workspace=workspace1, label="business", description="")
+        assert workspace1.updated != modified_datetime
+        assert workspace1.label == "business"
```

Comment:
this assert is useless: the label is the same in both revisions.

Reply:
removed and replace by more explicit test